.github/workflows/release.yaml

```
@@ -5,6 +5,9 @@ on:
     types:
       - published
 
+permissions:
+  contents: write
```

```suggestion
permissions:
  contents: read
```

The GitHub best practice is for following block to be added to the job while the workflow get least privilege.

```yaml
permissions:
  contents: write
```